build/dangerjs/checkPerfRegressions.ts

```diff
@@ -43,7 +43,12 @@ function fluentFabricComparision(danger, markdown, warn) {
   )
 
   const getStatus = fluentToFabric =>
-    fluentToFabric > 1 ? '🔧' : fluentToFabric >= 0.7 ? '🎯' : '🦄'
+    [
+      fluentToFabric <= 0.5 && '⚡',
+      fluentToFabric <= 0.7 && '🦄',
+      fluentToFabric <= 1 && '🎯',
+      '🔧',
```

Comment:
Our "target" is 30% faster.  The previous "on target' was displayed when a Fluent component was 30% faster than Fabric.

The 4 levels are fun, but it might make this harder to understand.

Reply:
Reverted 👍